src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,20 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot;
+
+/**
+ * Add your docs here.
+ */
+public class Constants {
+
+    public Constants() {
+        //Empty Constructer
+    }
+    public static final double TOP_GOAL_DEADBAND = 0.5;
```

Comment:
I don't think it's valuable to have this in this Constants file. It's very specific to the LimeLight top goal aim command.

Where I find a Constants file valuable is for all the CAN ids in the system.

Reply:
Yes, but we need that same deadband for both TopGoalCommand and vision.java.  Where would you store it?